apps/system/js/app_window.js

```diff
@@ -1278,7 +1278,7 @@
 
       this.screenshotOverlay.classList.remove('visible');
       this.screenshotOverlay.style.backgroundImage = '';
-      this.element.classList.remove('no-screenshot');
+      this.element && this.element.classList.remove('no-screenshot');
```

Comment:
did you mean for this change to be part of this patch? Or is this an unrelated change?

It seems like a safe change, and I'm fine with it as long as you meant for it to be here.